test/hotspot/jtreg/runtime/cds/appcds/TestDumpClassListSource.java

```diff
@@ -175,6 +176,10 @@ public static void main(String[] args) throws Exception {
         fileArchive.delete();
         fileList.delete();
 
+        if (!Platform.areCustomLoadersSupportedForCDS()) {
```

Comment:
It's better to use `@requires vm.cds.custom.loaders` in the test header.

Reply:
> It's better to use `@requires vm.cds.custom.loaders` in the test header.

Updated.
Thanks @iklam .